app/models/planet.py

```diff
@@ -0,0 +1,17 @@
+class Planet:
+    def __init__(self, id, name, description, distance_from_sun):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.distance_from_sun = distance_from_sun
+
+planets =[
```

Comment:
```suggestion
planets = [
```
Nit: missing whitespace after =